### DISCUSSION TITLE
Issue with underlying Borg - just with 1 profile out of 7

### DISCUSSION THREAD

**Discussion by jk-advedes** [borgbase/vorta#1720] [FAQ]
Dear all,
more than year ago, I gave setup Vorta to run daily backups (source: NAS shares, target: USB-attached HDD, all running on Rpi4).
For may months, everything run smoothly, but since mid of April, one of seven profiles is failing.
Checked:

- compared *.json of defective profile with working profiles (no difference except of intended)
- source as well as target are accessible from terminal
- no record in the Vorta - log file at ~/.cache/~/Vorta/log

Attached screenshot of Vorta with error displayed (traceback from Python, Borg directory).
Vorta version  0.8.10, Borg version 1.1.16., underlying OS: Debian bullseye 11 (Rpi-OS)
![Vorta-error](https://github.com/borgbase/vorta/assets/67338863/63e08096-c75c-4b4a-be5c-db05fe7298c1)
Any proposal welcome!

**Comment by real-yfprojects** [comment]
> * no record in the Vorta - log file at ~/.cache/~/Vorta/log

Pretty sure that all error messages output by borg appear in the logs.

**Comment by jk-advedes** [comment]
Sorry, 
I have found right log location; finally found the answer to my issue there: https://github.com/borgbackup/borg/issues/3912.
Deleting the cache helped.

**Comment by jk-advedes** [comment]
Issue resolved. Thx!
